Library/Formula/postgresql.rb

```diff
@@ -12,15 +13,15 @@ class Postgresql < Formula
     sha1 "efd661ace5a5f2657d047098dec15136aa4f0249" => :mountain_lion
   end
 
+  deprecated_option "no-perl" => "without-perl"
+  deprecated_option "no-tcl" => "without-tcl"
+  deprecated_option "enable-dtrace" => "with-dtrace"
```

Comment:
Done. I didn't know if the ` deprecated_options ` had to go above the current options to be valid.